Technical-Steering-Committee/issues.md

```diff
@@ -0,0 +1,42 @@
+# Issues
+
+All the squads within the Zowe projects use GitHub to track the issues raised against the projects
+they are responsible for. The issues cold be questions, bug reports, request for enhancements and others.
+Every squad responds to the new issues within the 90 days of the creation of the issue. 
+
+The squads can specify the details of the issue processing for specific project within the project
+repository in the ISSUES.md file in the root of the repository. 
+
+Every squad is expected to have at least one weekly meeting. The meeting should among other responsibilities
+triage the new issues and decide what to do with the issues. 
+
+The triage of the issues is done via attaching the labels to the issues. The recommended labels to use are:
+
+- new - The issue wasn't triaged yet 
+
+Types of the issues:
+
+- enhancement - Request to provide new functionality. [The recommended template](https://github.com/zowe/api-layer/blob/v2.x.x/.github/ISSUE_TEMPLATE/feature_request.md) 
+- question - Question around how something works in the specific projects. [The recommended template](https://github.com/zowe/api-layer/blob/v2.x.x/.github/ISSUE_TEMPLATE/question.md)
+- bug - Defect found in one of the existing functionalities within the project. [The recommended template](https://github.com/zowe/api-layer/blob/v2.x.x/.github/ISSUE_TEMPLATE/bug_report.md)
+
+Importance of the issue as perceived by squad organized by importance:
+
+- Priority: Critical - An issue that needs to be resolved within this PI, ideally within next sprint
+- Priority: High - An important issue that should be at least considered for the next PI, ideally within this one, if there is spare capacity
+- Priority: Medium - An issue that isn't very important so it will be planned in the PI, once there are no more important issues
+- Priority: Low - An issue that is recognized by the squad but that is not considered very important 
+
+When figuring out the next issue to work on the squad take into account the amount of thumbs-ups the users gave
+to the specific issue. The squad can find the information on the issues tab in the Github as shown on the figure 1.
+
+![Thumbs-up view](issues-thumbs-up.png "Figure 1: Thumbs-up view")
+
+The issues are organized into larger projects shown as epics on the Zenhub board whenever appropriate. The example of 
+epic within Zenhub is shown on the figure 2. 
+
+![Epic view](issues-epic.png "Figure 2: Epic view")
+
+The isues that are older than 180 days will be triaged at least once every PI unless they are associated with 
+existing Epic. The issues are either kept, linked to an epic or closed if they aren't considered relevant by the squad 
+anymore. 
```

Comment:
Curiosity:
- Will Milestones be considered an exception for not having to triage an old issue once every PI ?

Reply:
It's a good point, milestones should also be a valid pint.